### PR TITLE
added message template and variables

### PR DESCRIPTION
**Pull request by DFSko**

### CODE REVIEW

File: HappyTravel.StdOutLogger/Internals/StdOutLogger.cs [hunk 1]
```diff
@@ -114,6 +116,30 @@ private void AddScopedInformation(LogEntry logEntry, StringBuilder stringBuilder
         }
 
 
+        private static string GetMessageTemplate<TState>(TState state)
+        {
+            if (state is not IReadOnlyList<KeyValuePair<string, object>> values) return string.Empty;
```
Comment (mingazhev) [inline HappyTravel.StdOutLogger/Internals/StdOutLogger.cs:121]: On the next line

File: HappyTravel.StdOutLogger/Internals/StdOutLogger.cs [hunk 2]
```diff
@@ -114,6 +116,30 @@ private void AddScopedInformation(LogEntry logEntry, StringBuilder stringBuilder
...
+
+            return values.Where(v => v.Key == MessageTemplateKey)
+                .Select(v => v.Value)
+                .SingleOrDefault()?.ToString() ?? string.Empty;
+        }
+
+
+        private static void AddMessageVariables<TState>(LogEntry logEntry, TState state)
+        {
+            if (state is not IReadOnlyList<KeyValuePair<string, object>> values) return;
```
Comment (mingazhev) [inline HappyTravel.StdOutLogger/Internals/StdOutLogger.cs:131]: On the next line

File: HappyTravel.StdOutLogger/Internals/StdOutLogger.cs (hunk 1 above)
Comment (DFSko) [reply HappyTravel.StdOutLogger/Internals/StdOutLogger.cs]: Fixed

File: HappyTravel.StdOutLogger/Internals/StdOutLogger.cs (hunk 2 above)
Comment (DFSko) [reply HappyTravel.StdOutLogger/Internals/StdOutLogger.cs]: Fixed

File: HappyTravel.StdOutLogger/Internals/StdOutLogger.cs [hunk 3]
```diff
@@ -114,6 +116,32 @@ private void AddScopedInformation(LogEntry logEntry, StringBuilder stringBuilder
...
+        }
+
+
+        private static void AddMessageVariables<TState>(LogEntry logEntry, TState state)
+        {
+            if (state is not IReadOnlyList<KeyValuePair<string, object>> values) 
+                return;
+            
+            foreach (var (key, value) in values.Where(v => v.Key != MessageTemplateKey))
+            {
```
Comment (kirillta) [inline HappyTravel.StdOutLogger/Internals/StdOutLogger.cs:136]: Unnecessary braces

**Review by kirillta — APPROVED** [review]

File: HappyTravel.StdOutLogger/HappyTravel.StdOutLogger.csproj [hunk 4]
```diff
@@ -10,11 +10,11 @@
     <GeneratePackageOnBuild>true</GeneratePackageOnBuild>
```
Comment (kirillta) [inline HappyTravel.StdOutLogger/HappyTravel.StdOutLogger.csproj:10]: Also, please, add authors above

**Review by kirillta — APPROVED** [review]

File: HappyTravel.StdOutLogger/Internals/StdOutLogger.cs (hunk 3 above)
Comment (DFSko) [reply HappyTravel.StdOutLogger/Internals/StdOutLogger.cs]: Fixed

File: HappyTravel.StdOutLogger/HappyTravel.StdOutLogger.csproj (hunk 4 above)
Comment (DFSko) [reply HappyTravel.StdOutLogger/HappyTravel.StdOutLogger.csproj:10]: Authors already exists

**Pull request merged**